module_utils/network_lsr/argument_validator.py

```diff
@@ -1907,3 +1933,23 @@ def _ipv4_enabled(connection):
                     "Setting DNS options 'inet6', 'ip6-bytestring', 'ip6-dotint', "
                     "'no-ip6-dotint' is not allowed when IPv6 is disabled.",
                 )
+        # initscripts does not support match.path, so raise errors when network
+        # provider is initscripts
+        # When NM does not have attribute "SettingMatch" and "SETTING_MATCH_PATH",
+        # it will not support match.path, so raise errors when NM does not have
+        # attribute "SettingMatch" and "SETTING_MATCH_PATH"
```

Comment:
Please add the comments to each if clause (one for initscripts, one for NM).

Reply:
Updated.